salt/redash/upgrade-nointeraction

```diff
@@ -0,0 +1,236 @@
+#!/usr/bin/env python
+import urllib
+import argparse
+import os
+import subprocess
+import sys
+from collections import namedtuple
+from fnmatch import fnmatch
+
+import requests
+
```

Comment:
Would it be possible to order imports? Pycharm does it for you ;-) 'code/optimize imports'

Reply:
Added comments. It's not quite a full diff, but hopefully it's clear enough (and the source may change anyway, so I think this is better than a full diff)

Reply:
@odscjames sorry, I have just seen your message. I don't understand your answer. That you have copy/pasted something doesn't mean that you cannot fix something that is not ok. Would you leave a bug in a copy/paste code? I don't think so.

Reply:
Thanks for your explanation @odscjames .